### ISSUE TITLE
how can i predict and fine-tune using triplet loss models?

### ISSUE THREAD

**Issue by rlaals7349** [davidsandberg/facenet#1156]
i trained with triplet loss function( train_tripletloss.py ) with asain-dataset. 

i have 2 questions about triplet loss.

1. how can i predict( or test ) my image, using model trained with triplet loss?
all of test or prediction script is based on softmax activation function.

2. if it can, i want to fine-tine my model trained triplet loss with my dataset(around 20 images per one person, total 4 person). but all of script about fine-tuning is based on using softmax, so i can't find script i want using triplet loss. because i want to execute real_time_face_recognition.py with model trained triplet loss (softmax was done and it is work fine). 

Thanks.
